### DISCUSSION TITLE
Error Catching / Error Handling

### DISCUSSION THREAD

**Discussion by Erenedim** [typst/typst#3603] [Q&A]
How to handle error catching within a typst document?

My particular use case is inserting images from a list, if they exists, if they don't exist then do nothing.

```
#let list = ("A","B","C")

#for v in list [
  // how to catch and ignore error if image do not exist (path is invalid)
  #image("graphic/" + v + ".jpg")
]
```

I have tried to play around with #assert and #panic but to no avail.

Thanks in advance for the help! It is really appreciated!

**Comment by qujihan** [comment]
I want to know how to solve it too

**Comment by bluss** [comment]
There's a solution of that problem here https://sitandr.github.io/typst-examples-book/book/typstonomicon/try_catch.html

But it comes with disclaimers

- It is probably **not stable** (can stop working in future Typst)
- It smells *cursed*
